### PR TITLE
[Scalatra]Added gitignore generation function

### PR DESCRIPTION
**Pull request by magnolia-k**
Copy gitignore template from client

please review

@clasnake  @jimschubert  @shijinkui 

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Copy gitignore template from client

### CODE REVIEW

**Comment by jimschubert** [comment]
Looks good!

For anyone who comes across this PR... Usually I generate gitignores from gitignore.io (see kotlin client sample). This one with IntelliJ rather than Scala IDE can be generated here https://www.gitignore.io/api/sbt%2Cscala%2Cintellij

The .gitignore can be added to .swagger-codegen-ignore so regeneration doesn't overwrite .gitignore.

File: modules/swagger-codegen/src/main/resources/scalatra/gitignore.mustache [hunk 1]
```diff
@@ -0,0 +1,17 @@
+*.class
```
Comment (shijinkui) [inline modules/swagger-codegen/src/main/resources/scalatra/gitignore.mustache]: please add Intellij IDEA ignores:
```
.idea
*.iml
```

**Comment by wing328** [comment]
The CircleCI issue can be ignored. I'll review and merge this PR shortly.

**Comment by magnolia-k** [comment]
Thank you for the merge

As the content of gitignore seems to need to be added, I intend to issue another PR with client

**Pull request merged**